Makefile

```diff
@@ -6,7 +6,7 @@ UNAME_S := $(shell uname -s)
 
 # If you're using docker and docker-compose, you can use this Makefile to run
 # commands in your docker images by setting the DOCKER_PREFIX environment variable
-# to: DOCKER_PREFIX="docker-compose run --rm web"
+# to: DOCKER_PREFIX="docker-compose run --rm web --"
```

Comment:
Incidentally, we should really get rid of all of these `DOCKER_PREFIX` lines. I don't think they really help anyone, and they make things much slower by spawning a new container instance for each sub-command.